ui/css/itcss/components/send.scss

```
@@ -515,6 +515,11 @@
     margin-top: 5px;
   }
 
+  &__error-hex-data {
+    margin-top: 5px;
+    font-size: 0.7em;
```

As a default we should be using bounding box values(margin, padding) that fit into the 8px grid(multiples of 8 with the exception of `4px`) `4px, 8px, 16px, 24px`
Also we shouldn't be using any static font-size values in our css they should use provided mixins

```suggestion
    @include H7;

    margin-top: 4px;
```